whois-commons/src/main/java/net/ripe/db/whois/common/grs/AbstractAuthoritativeResourceLoader.java

```diff
@@ -0,0 +1,89 @@
+package net.ripe.db.whois.common.grs;
+
+import com.google.common.collect.Sets;
+import net.ripe.commons.ip.Asn;
+import net.ripe.commons.ip.AsnRange;
+import net.ripe.commons.ip.Ipv4;
+import net.ripe.commons.ip.Ipv4Range;
+import net.ripe.commons.ip.Ipv6;
+import net.ripe.commons.ip.Ipv6Range;
+import net.ripe.commons.ip.SortedRangeSet;
+import net.ripe.db.whois.common.ip.Ipv4Resource;
+import org.slf4j.Logger;
+
+import java.util.Set;
+
+abstract class AbstractAuthoritativeResourceLoader {
+
+    private final static Set<String> ALLOWED_STATUSES = Sets.newHashSet("allocated", "assigned", "available", "reserved");
```

Comment:
Use CIString? Or are these values always lowercase?

Reply:
They're part of delegated stats format and always lowercase apparently..

Reply:
Ok if it's already implemented this way